### PR TITLE
Add thumbnail cleanup cronjob.

### LINKED ISSUE #227: Thumbnails Cleanup Maintenance
oCIS currently does not offer a way to clean up thumbnails: https://github.com/owncloud/ocis/issues/6160

Therefore we should add a naive maintenance job that does just `rm -rf /var/lib/ocis/thumbnails`.

Configuration should look similar like:
https://github.com/owncloud/ocis-charts/blob/b9661665a79d7a4d53ddc14c5b9860b87cfdb778/charts/ocis/values.yaml#L972-L992

### PR DESCRIPTION
**Pull request by Excds**
## Description
Adding a CronJob that deletes thumbnails older than a configured amount of days.

## Related Issue
- Fixes #227 
- 
## How Has This Been Tested?
Tested with my local k3s setup.

### CODE REVIEW

File: charts/ocis/docs/values-desc-table.adoc [hunk 1]
```diff
@@ -1794,6 +1794,30 @@ a| [subs=-attributes]
...
++int+
+a| [subs=-attributes]
+`30`
+| Setting that makes the command delete all thumbnails older than the value. The value is a number in days.
+| services.thumbnails.maintenance.cleanUpOldThumbnails.enabled
+a| [subs=-attributes]
++bool+
+a| [subs=-attributes]
+`false`
+| Enables a job, that cleans up old thumbnails. Requires persistence to be enabled.
```
Comment (mmattel) [inline charts/ocis/docs/values-desc-table.adoc:1814]: What is meant with:
Requires persistence to be enabled
?

**Comment by mmattel** [comment]
Q: is the change in the .adoc table "handcraftet" or generated?
I have somthing in my mind that the source is another file and the .adoc file is generated from it - but I may be wrong...

**Comment by Excds** [comment]
> Q: is the change in the .adoc table "handcraftet" or generated?
> I have somthing in my mind that the source is another file and the .adoc file is generated from it - but I may be wrong...

It is generated. And this _should_ be moved to CI build or something. But when we have the time.™

File: charts/ocis/templates/thumbnails/jobs.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: thumbnails-cleanup-script-configmap
+data:
+  thumbnails-cleanup-script.sh: |
+    #!/bin/sh
+
+    find /var/lib/ocis/thumbnails -atime {{ .Values.services.thumbnails.maintenance.cleanUpOldThumbnails.deleteBefore }} -exec rm -rf {} \;
```
[the remark this answers is not in the document]
Comment (Excds) [reply charts/ocis/templates/thumbnails/jobs.yaml]: Fixed.

File: charts/ocis/docs/values-desc-table.adoc (hunk 1 above)
Comment (mmattel) [reply charts/ocis/docs/values-desc-table.adoc:1814]: > if a pod gets deleted then data will also be deleted

ah, ok, then what about:
`Only recommended if pod persistence is setup.`

Comment (mmattel) [reply charts/ocis/docs/values-desc-table.adoc:1814]: What about that we describe what persistence means in the context of helm charts in the admin documentation?

**Pull request merged**